packages/flutter_tools/lib/src/build_system/targets/ios.dart

```diff
@@ -373,16 +370,14 @@ abstract class UnpackIOS extends Target {
     }
   }
 
-  /// Destructively strip bitcode from the framework, if needed.
+  /// Destructively strip bitcode from the framework. This can be removed
+  /// when the framework is no longer built with bitcode.
   void _bitcodeStripFramework(Environment environment, String frameworkBinaryPath) {
-    if (environment.defines[kBitcodeFlag] == 'true') {
-      return;
-    }
     final ProcessResult stripResult = environment.processManager.runSync(<String>[
       'xcrun',
       'bitcode_strip',
       frameworkBinaryPath,
-      '-m', // leave the bitcode marker.
+      '-r', // Delete the bitcode segment.
```

Comment:
Just made this change, totally delete it instead of leaving the marker:
```
OPTIONS
       -r     Remove the __LLVM bitcode segment entirely.

       -m     Remove the bitcode from the __LLVM segment, leaving behind a
              marker.
```